packages/testing/playwright/pages/CanvasPage.ts

```diff
@@ -1,11 +1,19 @@
-import type { Locator } from '@playwright/test';
+import type { Locator, Page } from '@playwright/test';
 import { nanoid } from 'nanoid';
 
 import { BasePage } from './BasePage';
 import { ROUTES } from '../config/constants';
 import { resolveFromRoot } from '../utils/path-helper';
+import { LogsPanel } from './parts/LogsPanel';
 
 export class CanvasPage extends BasePage {
+	readonly logsPanel: LogsPanel;
```

Comment:
Thanks, I was talking to Artem and this does seem like the best approach we can take for some of the more complicated pages.